modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -183,8 +191,11 @@ def get_first_name_from_form_data
         form20_10207_contact_info[1]
       when 'vba_40_0247'
         form_data.dig('applicant_full_name', 'first')
+      when 'vba_40_10007'
+        form_data.dig('application', 'claimant', 'name', 'first')
```

Comment:
@ksantiagoBAH Sorry if I missed it, but is this being tested anywhere?

Reply:
I have some test in place for the notification email for the 40-10007, do you know where in the other lines of code above are being tested?

Reply:
NVM